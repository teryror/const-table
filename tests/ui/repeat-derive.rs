use const_table::const_table;

#[const_table]
#[derive(Debug)]
enum SpeciesID {
    SpeciesInfo { pub sound: &'static str },
    Human = SpeciesInfo { sound: "Hello World" }
}

fn main() {}