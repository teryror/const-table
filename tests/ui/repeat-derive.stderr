error: the Debug trait is already implemented by the const_table macro
 --> $DIR/repeat-derive.rs:4:10
  |
4 | #[derive(Debug)]
  |          ^^^^^
